pyecobee/__init__.py

```diff
@@ -259,6 +259,7 @@ def set_fan_mode(
         cool_temp: int,
         heat_temp: int,
         hold_type: str = "nextTransition",
+        hold_hours: str = "2",
```

Comment:
Do we need to make hold_hours a kwarg with a default other than None? I'm now thinking the same thing for hold_type, now that Home Assistant passes in the actual set value rather than always nextTransition. If the hold_type is not "holdHours", we don't need to pass hold_hours, and if it is, then the calling method in Home Assistant should be passing in hold_hours.